backend/app/api/transactions.py

```
@@ -115,14 +125,45 @@ async def import_transactions(
+                    if CryptoCSVParser.is_crypto_transaction(ticker):
+                        isin = CryptoCSVParser.generate_crypto_identifier(ticker, description)
+                        logger.info(f"Generated crypto ISIN {isin} for ticker {ticker}")
+                    else:
+                        # For traditional assets, try to fetch ISIN from Yahoo Finance
+                        try:
+                            fetched_isin, fetched_description = await fetch_ticker_metadata(ticker)
+                            isin = fetched_isin
+                            if fetched_description and not description:
+                                description = fetched_description
+                            logger.info(f"Fetched ISIN {isin} for ticker {ticker}")
+                        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)